plugins/org.jboss.tools.openshift.ui/src/org/jboss/tools/openshift/internal/ui/preferences/SSLCertificatesPreference.java

```diff
@@ -105,13 +106,19 @@ public void setAllowedByCertificate(X509Certificate certificate, boolean result)
 	private boolean isValid(String validity) {
 		int expiresOnIndex = validity.indexOf(X509CertificateParser.EXPIRES_ON_PREFIX);
 		if(expiresOnIndex >= 0) {
-			SimpleDateFormat dateFormat = new SimpleDateFormat(X509CertificateParser.DATE_FORMAT);
+			SimpleDateFormat dateFormat = new SimpleDateFormat(X509CertificateParser.DATE_FORMAT,  Locale.ENGLISH);
```

Comment:
@robstryker @scabanovich Could you please update this so that we can include this change in the upcoming 4.4.2.AM2?